archinstall/lib/installer.py

```diff
@@ -1186,7 +1187,7 @@ def _add_grub_bootloader(
 				boot_dir = boot_partition.mountpoint
 
 			add_options = [
-				'--target=x86_64-efi',
+				f'--target={platform.machine()}-elf',
```

Comment:
With an x86_64 system this produces a target value of `x86_64-elf` but according to [grub-install(8)](https://man.archlinux.org/man/core/grub/grub-install.8.en#target) it should be `x86_64-efi`. There are no target values with the suffix `-elf`, is this a typo?

Reply:
Oh, sorry.